Tools/_sunrise/prepare_test_shards.py

```
@@ -0,0 +1,76 @@
+        [
+            "dotnet",
+            "test",
+            "--list-tests",
+            "--no-build",
+            "--no-restore",
+            "--configuration",
+            "DebugOpt",
+            "Content.IntegrationTests/Content.IntegrationTests.csproj",
+            "--",
+            "NUnit.DisplayName=FullName",
+        ],
```

Ruff (0.15.20)

[error] Starting a process with a partial executable path

(S607)